docker-compose.yml

```diff
@@ -10,9 +20,9 @@ services:
     ports:
     - 8080:8080
     - 4000:4000
+    - 4001:4001
```

Comment:
This has been a problem before, but since we have a port that's supposed to be firewalled we should probably start prefixing `ports` with `127.0.0.1:`, so this would become `127.0.0.1:4001:4001` and we should probably do the same for the other ports.

Reply:
ok will do